### PR TITLE
ci: Fix fetch github command

### LINKED ISSUE #120: Fix fetch command line
Fix it for the fetch branches tool

### PR DESCRIPTION
**Pull request by GabyCT**
ci: Fix command line for the fetch of the branch.

Fixes #120

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: cmd/fetchbranches/main.go [hunk 1]
```diff
@@ -99,9 +99,9 @@ func fetchBranches(fetchMap map[string]string, pr pr) error {
 		fetchDirectory := filepath.Join(fullDirectory, pr.owner, branchRepo)
 		os.Chdir(fetchDirectory)
 
-		cmd = exec.Command("git", "fetch", "origin")
-		nameFetch := filepath.Join("origin", branchName)
-		cmd = exec.Command("git", "checkout", "-b", branchName, nameFetch)
+		// Brings the change of the PR
+		nameFetch := fmt.Sprintf("pull/%d/head", pr.number)
```
Comment (devimc) [inline cmd/fetchbranches/main.go]: this won't work

Comment (devimc) [inline cmd/fetchbranches/main.go]: you have to use the url defined in the tag

to pull the branch of a user you could use
```
 git pull https://github.com/$USER/$REPO $BRANCH  --no-edit
```

for example
```
git pull https://github.com/GabyCT/tests topic/fixfetch  --no-edit
```

**Comment by GabyCT** [comment]
@devimc changes applied

**Comment by GabyCT** [comment]
@devimc thanks for the feedback

**Comment by chavafg** [comment]
lgtm

**Pull request merged**